integration-tests/cstor-pool/cstor_integration_test.go

```diff
@@ -0,0 +1,370 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstorpoolit
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	csp "github.com/openebs/maya/pkg/cstorpool/v1alpha3"
+	spc "github.com/openebs/maya/pkg/storagepoolclaim/v1alpha1"
+	"time"
+
+	//	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"testing"
+)
+
+const (
+	KubeConfigPath = "/home/ashutosh/.kube/config"
```

Comment:
nit: check the path ^^

Reply:
Yes ... We need to finalise on the path. Just pushed for reference. It is for my local setup.
Let us come up how we may want to run tests. 
cc @AmitKumarDas